man/systemd-pcrlock.xml

```diff
@@ -155,6 +155,19 @@
         <para>If the new prediction matches the old this command terminates quickly and executes no further
         operation. (Unless <option>--force</option> is specified, see below.)</para>
 
+        <para>A copy of the <filename>/var/lib/systemd/pcrlock.json</filename> policy file is encoded in a
```

Comment:
Would be good to mention the version here

Reply:
version of what?

Reply:
the version this change was added in, given pcrlock shipped in v255 but this functionality is important enough and will be in 256